src/ds/thread_messaging.h

```diff
@@ -224,94 +224,122 @@ namespace threading
       }
     }
 
-  public:
-    static ThreadMessaging thread_messaging;
-    static std::atomic<uint16_t> thread_count;
-    static const uint16_t main_thread = MAIN_THREAD_ID;
+    inline TaskQueue& get_tasks(uint16_t task_id)
```

Comment:
Should `task_id` be `thread_id` here, as in "get the tasks for this thread"?

Reply:
I'd like to move towards separating this job-system task-queue from thread affinity, and to think of this as a fixed-size task-queue collection, keyed by `task_id`. We happen to maintain 1 queue-per-thread, but that's handled elsewhere. Admittedly, this is a soft and poorly maintained distinction, not least because this whole "job system" is called `ThreadMessaging`, but the renames here are an attempt to move in that direction.